### PR TITLE
Add some more tests for DocumentDB illegal characters checking, replacement

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
As per title: getting used to the codebase, so adding one new test (replacement wasn't being tested), expand existing one.

### CODE REVIEW

**Comment by tatu-at-datastax** [comment]
Hmmh. Long-running integration tests failing on formatting checks... annoying. Esp. as those logs are big.
Next need to figure out how to change formatting (it is in test file I changed), presumably some IntelliJ setting could do it).

**Comment by tatu-at-datastax** [comment]
Ready for review, after fixing formatting.

File: restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java [hunk 1]
```diff
@@ -83,14 +83,57 @@ public void getForbiddenCharactersMessage() {
...
   public void containsIllegalChars() {
-    assertThat(DocumentDB.containsIllegalChars("[")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("]")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars(",")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars(".")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("\'")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("*")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("a[b")).isTrue();
-    assertThat(DocumentDB.containsIllegalChars("\"")).isFalse();
+    // first some positive matches
```
Comment (mpenick) [inline restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java:86]: These might be a good candidate for a parameterized test.

```java
  @ParameterizedTest
  @ValueSource(strings = {"[", "]", ... })
  public void containsIllegalChars(String str) {
     // ...
```

Comment (tatu-at-datastax) [reply restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java:86]: Good point, I can change that.

Comment (tatu-at-datastax) [reply restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java]: Changed, split into couple of separate test methods to keep more manageable.

**Review by mpenick — APPROVED** [review]

**Comment by tatu-at-datastax** [comment]
Looks like this will be obsolete by encoding changes... :)

**Review by dimas-b — APPROVED** [review]

**Comment by tatu-at-datastax** [comment]
Obsoleted by other changes, closing.

**Pull request closed without merging**